.github/actions/update-repo-description/pick.py

```
@@ -0,0 +1,139 @@
+		lmd = lunar_md(today)
+		if lmd and lmd in lunar_dates:
+			return True
+	terms = holiday.get('jieqi') or ()
+	if terms:
+		jq = jieqi_name(today)
+		if jq and jq in terms:
+			return True
+	yearday = holiday.get('yearday')
+	if yearday is not None and today.timetuple().tm_yday == yearday:
+		return True
+	return False
```

Ruff (0.16.1)

[warning] Return the condition directly

Inline condition

(SIM103)